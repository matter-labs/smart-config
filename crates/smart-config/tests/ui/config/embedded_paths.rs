fn main() {
    smart_config::config!("test": 1, "test.path": 2);
}
