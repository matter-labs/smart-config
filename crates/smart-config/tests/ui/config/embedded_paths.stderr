error[E0080]: evaluation of constant value failed
 --> tests/ui/config/embedded_paths.rs:2:5
  |
2 |     smart_config::config!("test": 1, "test.path": 2);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the evaluated program panicked at 'Path #0 `test` is a prefix of path #1 `test.path`', $DIR/tests/ui/config/embedded_paths.rs:2:5
  |
  = note: this error originates in the macro `smart_config::config` (in Nightly builds, run with -Z macro-backtrace for more info)
